packages/terra-paginator/src/ProgressivePaginator.jsx

```diff
@@ -47,6 +47,14 @@ const propTypes = {
    * Allows user to set custom page label. _(usage note: User must pass translated text)_.
    */
   pageLabel: PropTypes.string,
+  /**
+   * Apply Fusion anchor style or not.
+   */
+  fusionAnchor: PropTypes.bool,
```

Comment:
It seems odd to add a prop for a very specific use case. Especially since it's related to theming. I'm confused why this is needed to be an API change and not a theme update or any other alternative solution.

cc: @JessieRandle , @mjpalazzo , @sycombs

Reply:
For the Orion fusion theme, there are 2 different styles(anchor and canvas) and we need to support both. So applying this new optional prop is to distinguish each other in the same fusion theme.
<img width="1330" alt="Screenshot 2023-09-13 at 1 09 41 PM" src="https://github.com/cerner/terra-core/assets/44185348/8e6e56f0-39ca-4644-bfae-d9beb8fcc94c">
Any suggestions?

Reply:
If it's styling we need to support then I would say that we should support it for all themes and not add a niche prop that is only useful for a specific theme. 

cc: @eawww , @mjpalazzo , @JessieRandle